metadata.yml

```diff
@@ -11,11 +16,14 @@ annotations:
   io.artifacthub.displayName: Verify Image Signatures
   io.artifacthub.resources: Pod
   io.artifacthub.keywords: pod, signature, sigstore, trusted
+  # kubewarden specific
   io.kubewarden.policy.ociUrl: ghcr.io/kubewarden/policies/verify-image-signatures
-  # rest
   io.kubewarden.policy.title: verify-image-signatures
-  io.kubewarden.policy.description: A Kubewarden Policy that verifies all the signatures of the container images referenced by a Pod
-  io.kubewarden.policy.author: "Kubewarden developers <cncf-kubewarden-maintainers@lists.cncf.io>"
+  io.kubewarden.policy.description: A Kubewarden Policy that verifies all the signatures
+    of the container images referenced by a Pod
+  io.kubewarden.policy.author: Kubewarden developers <cncf-kubewarden-maintainers@lists.cncf.io>
   io.kubewarden.policy.url: https://github.com/kubewarden/verify-image-signatures
   io.kubewarden.policy.source: https://github.com/kubewarden/verify-image-signatures
   io.kubewarden.policy.license: Apache-2.0
+  io.kubewarden.policy.category: Secure supply chain
+  io.kubewarden.policy.severity: medium
```

Comment:
could be `high`, because it's a precondition for other things, but I'm ok also with `medium`.